### PR TITLE
Add graph-safe reference 6.3.0

### PR DESCRIPTION
**Pull request by neon60**
Add reference to the graph-safe section.

### CODE REVIEW

**Review by amd-jnovotny — APPROVED** [review]
So it looks like some material from develop needed to be added here? Does it need to be added to the 6.3 release branch too? Perhaps check with the dev team and get their approval to ensure nothing gets missed.

**Comment by neon60** [comment]
> So it looks like some material from develop needed to be added here? Does it need to be added to the 6.3 release branch too? Perhaps check with the dev team and get their approval to ensure nothing gets missed.

Asked the dev team. Waiting for response.

**Review by evetsso — APPROVED** [review]

**Pull request merged**
